graphql2/schema.graphql

```diff
@@ -972,7 +973,11 @@ input CalcRotationHandoffTimesInput {
   handoff: ISOTimestamp!
   from: ISOTimestamp
   timeZone: String!
-  shiftLengthHours: Int!
+
+  # Deprecated shiftLength as an integer of length in hours
+  shiftLength: Int
```

Comment:
We can't change the existing field name as that would be a breaking change, this needs to remain `shiftLengthHours`

Make the new field `shiftLength` since units are included with it's type